### DISCUSSION TITLE
You are using a non-standard "NODE_ENV" value in your environment. This creates inconsistencies in

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#18253] [Help] [answered]
in `package.json`

```
  "scripts": {
    "test1": "cross-env NODE_ENV=test1 PORT=3000 node node.js",
    "test2": "cross-env NODE_ENV=test2 PORT=3000 node node.js",
    "build": "next build",
    "export": "next build && next export",
    "start": "next start"
  },
```

in `a.tsx`
```
const NODE_ENV = process.env.NODE_ENV; 
const  url = NODE_ENV+"/login"

```

i got the error:  
` You are using a non-standard "NODE_ENV" value in your environment. This creates inconsistencies in `

-----

**Comment by ghost** [comment, marked as answer]
solution:

change `NODE_ENV` to other,   

i change `NODE_ENV` to `APP_ENV`,  but not work

must change to      `NEXT_PUBLIC_APP_ENV`,   

the point **NEXT_PUBLIC_**

**Comment by Evavic44** [comment]
Thanks :) Also works with the CLI command: 

```js
setx NODE_ENV NEXT_PUBLIC_APP_ENV
```
